### PR TITLE
app tile clickability on tablet (bug 1118367)

### PR DESCRIPTION
**Pull request by spasovski**
Pretty self explanatory.

### CODE REVIEW

**Comment by ngokevin** [comment]
what's covering it

**Comment by spasovski** [comment]
Honestly not sure though clearly something not covered in a media query. Layering rules can be tricky http://philipwalton.com/articles/what-no-one-told-you-about-z-index/

File: src/media/css/tiles.styl [hunk 1]
```diff
@@ -20,6 +20,7 @@ $greater-than-mobile = unquote('(min-width: 330px)');
     display: block;
     overflow: hidden;
     position: relative;
+    z-index: 1;  // Any other overlay can overlap.
```
Comment (ngokevin) [inline src/media/css/tiles.styl]: link to the bug in this comment

**Comment by ngokevin** [comment]
r+wc

**Comment by spasovski** [comment]
Pushed via direct commit with comments addressed.

**Pull request closed without merging**